packages/redis_client/lib/src/redis_client.dart

```diff
@@ -260,21 +376,17 @@ class RedisJson {
     required String key,
     required Map<String, dynamic> value,
   }) {
-    return _client.sendCommand(['JSON.SET', key, r'$', json.encode(value)]);
+    return _client.execute(['JSON.SET', key, r'$', json.encode(value)]);
   }
 
   /// Gets the value of a key.
   /// Returns null if the key does not exist.
   /// Equivalent to the `JSON.GET` command.
   /// https://redis.io/commands/json.get
   Future<Map<String, dynamic>?> get({required String key}) async {
-    final result = await _client.sendCommand([
-      'JSON.GET',
-      key,
-      r'$',
-    ]);
-    if (result is RespBulkString) {
-      final parts = LineSplitter.split(result.payload ?? '');
+    final result = await _client.execute(['JSON.GET', key, r'$']);
```

Comment:
Thoughts on extracting these json method names to consts?

Reply:
They should only be used in this one spot so I didn't see as much value but happy to do so if you prefer.